### ISSUE TITLE
Files missing from package

### ISSUE THREAD

**Issue by ricardomatias** [mattjennings/svelte-pixi#8]
The following files are imported by dist/index.js but they are not bundled with the package.

```
Error: Build failed with 10 errors:
node_modules/svelte-pixi/dist/index.js:1:42: error: Could not resolve "./AnimatedSprite.svelte"
node_modules/svelte-pixi/dist/index.js:2:37: error: Could not resolve "./Container.svelte"
node_modules/svelte-pixi/dist/index.js:3:34: error: Could not resolve "./Loader.svelte"
node_modules/svelte-pixi/dist/index.js:4:34: error: Could not resolve "./Sprite.svelte"
node_modules/svelte-pixi/dist/index.js:5:40: error: Could not resolve "./TilingSprite.svelte"
```

**Comment by mattjennings** [comment]
oops, looks like something went wrong with the 0.0.3 release. Thanks for reporting this, I'll take a look!

**Comment by mattjennings** [comment]
Fixed in 0.0.4
